server/claim-revision/schema/claim-revision.schema.ts

```diff
@@ -2,15 +2,10 @@ import { Prop, Schema, SchemaFactory } from "@nestjs/mongoose";
 import * as mongoose from "mongoose";
 import { Claim } from "../../claim/schemas/claim.schema";
 import { Personality } from "../../personality/schemas/personality.schema";
+import { ContentModelEnum } from "types/enums";
```

Comment:
**issue (blocking):** this is breaking the build, I believe you should use a relative path.